_includes/help/nav_sidenav.html

```diff
@@ -11,7 +11,7 @@
         {% assign is_initially_expanded = 'false' %}
         {% assign is_initially_hidden = 'true' %}
       {% endif %}
-      <li class="usa-sidenav__item usa-parent" aria-expanded="{{ is_initially_expanded }}">
```

Comment:
Can we remove the assignment of the `is_initially_expanded` variable above if it's now unused?